Makefile

```diff
@@ -12,7 +12,4 @@ init: ## Initialize Terraform, on first-use or when adding modules or plugins.
 	@git config core.hooksPath .githooks
 
 format: ## Format your code automatically.
-	@terraform fmt
-
-plan: ## See how your changes would affect our infrastructure if applied.
-	@terraform plan
```

Comment:
We run this per-workspace now (from each directory, rather than the root), so we'll just use the CLI directly.